README.md

```diff
@@ -96,6 +96,19 @@ client = Octokit::Client.new(:login => 'defunkt', :password => 'c0d3b4ssssss!')
 client.user
 ```
 
+### Additional Query Parameters
+
+When passing additional parameters to GET based request use the following syntax:
+
+```ruby
+ # query: { parameter_name: 'value' }
+ # Example: Get repository listing by owner in ascending order
+ client.repos({}, query: {type: 'owner', sort: 'asc'})
+
+ # Example: Get contents of a repository by ref
+ client.contents(repo: 'octokit/octokit.rb', path: 'path/to/file', query: {ref: 'some-other-branch'})
+```
```

Comment:
Should we have some sort of file extension on `path/to/file` to make sure people don't mistake this for the containing directory?

Also, how do you feel about adding the resulting URL, just to cement exactly what we mean by query parameters?

```
https://api.github.com/repos/octokit/octokit.rb/contents/path/to/file?ref=some-other-branch
```

Reply:
Appreciate the feedback and I've added the changes !